frontend/src/metabase/visualizations/echarts/pie/model/index.ts

```diff
@@ -343,18 +343,21 @@ export function getPieChartModel(
 
   // Create sliceTree, fill out the innermost slice ring
   const sliceTree: SliceTree = new Map();
+  let i = 0;
   const [sliceTreeNodes, others] = _.chain(pieRowsWithValues)
-    .map(({ value, color, key, name, isOther }, index) => {
+    .map(({ value, color, key, name, isOther }) => {
       const visible = isOther
         ? !hiddenSlices.includes(OTHER_SLICE_KEY)
         : !hiddenSlices.includes(key);
 
-      return {
+      const normalizedPercentage = visible ? value / total : 0; // slice percentage values are normalized to 0-1 scale
+
+      const o = {
```

Comment:
nit: we could give it a name that'd tell that is, maybe `slice`?

Reply:
Woops, I saw your comment after merging 🤦‍♂️